auth/temporary-credentials/index.md

```diff
@@ -51,15 +46,47 @@ construction of the `temporaryAccessToken`, as detailed later. The `signature`
 property validates the authenticity of the certificate and is generated as
 detailed below.
 
+Named Temporary Credentials
+---------------------------
+
+It is often useful to have the clientId indicate something about the user
+responsible for the API call.  For example, the clientId `tc-login` issues
+temporary credentials to users, but including `"tc-login"` in a logfile is not
+of much use.  By specifying `tc-login` as issuer, this service can create
+temporary credentials with more descriptive clientIds, such as
+`tc-login/sso/someuser@mozilla.com`.
+
+This is accomplished by including the actual cliendId used to issue the
+temporary credentials in the  `issuer` property of the certificate, along with
+a `name` property giving the name that should be used with the credential.  The
+issuer must have the scope `auth:create-client:<name>`, and any use of the
+credential must use `name` as the `clientId`.
+
 Signature Generation
 --------------------
 The signature for a certificate is generated by the issuer, the client that
 issues the temporary credentials. The issuer does this by generating a
 base64 encoded sha256 HMAC signature of the following string using the
-issuers `accessToken`.
+issuer's `accessToken`.
+
+```
+version:1\n
+seed:<seed>\n
+start:<start>\n
+expiry:<expiry>\n
+scopes:\n
+<scopes[0]>\n
+<scopes[1]>\n
+...
+<scopes[n]>
+```
+
+or, for named temporary credentials:
 
 ```
 version:1\n
+name:<name>\n
+issuer:<issuer>\n
```

Comment:
Ah, can we call name for `impersonate:` or `clientId:` ? or maybe even better `identity: ...`


----
I'm not sure signing issuer is a good idea.. Just because I see no value from it... so it confuses... okay, maybe I'm too pure here...